.github/workflows/version-bump.yml

```diff
@@ -89,73 +80,11 @@ jobs:
       # VERSION BUMP SECTION #
       ########################
 
-      ### napi
-      - name: Bump @bitwarden/sdk-napi Version
-        if: ${{ inputs.project == 'napi' }}
-        working-directory: "crates/bitwarden-napi"
-        run: |
-          npm ci
-          npm version ${{ inputs.version_number }}
-
-      - name: Bump napi crate Version
-        if: ${{ inputs.project == 'napi' }}
-        run: cargo set-version -p bitwarden-napi ${{ inputs.version_number }}
-
       ### bitwarden
       - name: Bump bitwarden crate Version
         if: ${{ inputs.project == 'bitwarden' }}
         run: cargo set-version -p bitwarden ${{ inputs.version_number }}
```

Comment:
We don't have a `bitwarden` crate anymore do we?

Reply:
We don't fixed.